client/Window.cpp

```diff
@@ -179,27 +179,41 @@ bool Window::initializeObjects()
 	//  ==========  Character Initialization   ========== 
 
 	// Characters on the map now (scaled 3x)
+	// chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
+	// 	glm::vec3(5.f, 1.f, 5.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
+	// 	"shaders/character/MAGE"));	
+	// chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
+	// 	glm::vec3(15.f, 1.f, 5.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
+	// 	"shaders/character/MAGE"));	
+	// chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
+	// 	glm::vec3(5.f, 1.f, 15.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
+	// 	"shaders/character/MAGE"));	
+	// chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
+	// 	glm::vec3(70.f, 1.f, 70.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
+	// 	"shaders/character/MAGE"));
+
+	// #ifdef SERVER_ENABLED
+	// clientChar = chars[client->getId()];
+
+	// #else 
+	// clientChar = chars[0];
+	// gameStarted = true;
+	// #endif
+
+	#ifndef SERVER_ENABLED
 	chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
 		glm::vec3(5.f, 1.f, 5.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
 		"shaders/character/MAGE"));	
-	chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
-		glm::vec3(15.f, 1.f, 5.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
-		"shaders/character/MAGE"));	
-	chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
-		glm::vec3(5.f, 1.f, 15.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
-		"shaders/character/MAGE"));	
-	chars.push_back(new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
-		glm::vec3(70.f, 1.f, 70.f), glm::vec3(0.f, 1.f, 0.f), glm::radians(0.f), 5.f, glm::vec3(1.f, .5f, .5f),
-		"shaders/character/MAGE"));
-
-	#ifdef SERVER_ENABLED
-	clientChar = chars[client->getId()];
-
-	#else 
 	clientChar = chars[0];
-	gameStarted = true;
+	Window::gameStarted = true;
 	#endif
 
+	chars[3] = (new Character("shaders/character/billboard.obj", &projection, &view, &eyePos, texShader,
```

Comment:
Monster (or chars[3]) should have SPAWN_POSITIONS[3][0] for x and SPAWN_POSITIONS[3][1] for z.

In other words, replace 115.f with SPAWN_POSITIONS[3][0] and 5.f with SPAWN_POSITIONS[3][1] in the translation vector (first vec3 of the constructor).